sql/opt_subselect.cc

```diff
@@ -5801,9 +5805,14 @@ static Item *remove_additional_cond(Item* conds)
   Remove the predicates pushed down into the subquery
 
   SYNOPSIS
-    remove_subq_pushed_predicates()
-      where   IN  Must be NULL
-              OUT The remaining WHERE condition, or NULL
+    remove_subq_pushed_predicates_impl()
+
+    @where   IN  If not NULL and predicate eligible for removal, this function
```

Comment:
`@where` ? I think the right doxygen format is `@param` ...

The relationship between `item` and `*where` is not clear.  does `item==*where` or not?